packages/ui/package.json

```diff
@@ -1,5 +1,31 @@
 {
   "name": "@cms/ui",
   "version": "0.1.0",
-  "private": true
+  "private": true,
+  "exports": {
```

Comment:
This is how we explicitly state multiple entrypoints to a package. You'll now be able to use imports like:

- `import { Accordion } from '@cms/ui/accordion'`
- `import { AlertDialog } from '@cms/ui/alert-dialog'`

And so on.

Reply:
Okay, this one looks quality of life thing from package.json itself. 
Learned something new here, haha.